web/styles/zulip.css

```diff
@@ -3311,3 +3311,12 @@ select.invite-as {
         }
     }
 }
+
+.keyboard-shortcut-option {
+    & span.tooltip-hotkey-hint {
+        margin-left: auto;
+        padding: 0 4px;
+        color: hsl(227deg 78% 59%);
+        border-color: hsl(227deg 78% 59%);
+    }
+}
```

Comment:
Let's exract these color values so that we don't need to touch `dark_theme.css` for this.